events/Lambda/AddPermission20150331v2.json

```diff
@@ -9,6 +9,35 @@
     "mitreAttackTechniques": [
         "T1098 - Account Manipulation"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1098 - Account Manipulation",
+            "reason": "The AddPermission API call can be used to alter permissions, effectively manipulating accounts to maintain access or escalate privileges."
+        },
+        {
+            "technique": "T1070 - Indicator Removal",
+            "reason": "Using AddPermission to adjust access rights can help hide malicious activities by removing or altering indicators of compromise."
+        },
+        {
+            "technique": "T1040 - Credential Access via API",
+            "reason": "Granting permissions can facilitate the capture or misuse of credentials if permissions are set incorrectly, allowing access to sensitive functions."
```

Comment:
I'm not really sure about this one, also T1040 is Network Sniffing https://attack.mitre.org/techniques/T1040/

Reply:
I do not know what went in my head. I'll remove it.